processes/README.md

```diff
@@ -8,6 +8,11 @@ This folder contains user-defined processes that can be run by any openEO client
 | -- | ---------- | ------- |
 | [array_contains_nodata](array_contains_nodata.json) | arrays | Check for no-data values in an array |
 | [array_find_nodata](array_find_nodata.json)         | arrays | Find no-data values in an array |
+| [anomaly_detection](anomaly_detection.json)         | arrays | Regional Benchmarking using CropSAR |
```

Comment:
The categories seem wrong. Can you update these to the categories used in the process files?

Reply:
I updated it to DataArray, but I not sure that if is that was what is to be specified as categories